package fi.lipp.blog.model.exceptions

class DiaryNotFoundException : BlogException("Diary not found", 404)
